### PR TITLE
Performance v1

### PR DESCRIPTION
**Pull request by MartijnReeuwijk**
Updated web app standards, SEO, accessibility and performance.
The site will show an offline page when you do not have internet.

### CODE REVIEW

**Comment by MartijnReeuwijk** [comment]
Fix #32

File: static/worker.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+var CACHE_NAME = 'semia-cache-v1';
```
Comment (jeroentvb) [inline static/worker.js]: cont

File: views/error.ejs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <%- include('./partials/head'); -%>
+<link rel="stylesheet" type="text/css" href="/css/errorpage.css">
```
Comment (jeroentvb) [inline views/error.ejs]: Move the css from the file into a `<style>` element

File: views/offline.ejs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<%- include('./partials/head'); -%>
+<link rel="stylesheet" type="text/css" href="/css/errorpage.css">
```
Comment (jeroentvb) [inline views/offline.ejs]: Move the css from the file into a `<style>` element

**Pull request merged**